### DISCUSSION TITLE
Translatable ?

### DISCUSSION THREAD

**Discussion by mondlicht-und-sterne** [zyrouge/symphony#6] [General]
Is the app translatable and when, where are the  strings to translate ?
Thanks in advance.

**Comment by zyrouge** [comment]
It is possible. Refer [this](https://github.com/zyrouge/symphony/blob/02d9cdce9727f104d0c32930735eeabeaaa61b9d/app/src/main/java/io/github/zyrouge/symphony/services/i18n/Translations.kt#L87).

**Reply by MichelangeloDePascale02** [reply]
Hello! First off, great job with your app. I was looking for a Music Player to listen to music offline that was simple and stable, and for now I can't complain at all with Symphony! I'd be interested in translating the application strings into Italian. I've developed a few Android apps in the past, but I've never tried using Kotlin. Could you explain to me how the file you mentioned works? Thanks and keep it up!

**Reply by zyrouge** [reply]
> Hello! First off, great job with your app. I was looking for a Music Player to listen to music offline that was simple and stable, and for now I can't complain at all with Symphony! I'd be interested in translating the application strings into Italian. I've developed a few Android apps in the past, but I've never tried using Kotlin. Could you explain to me how the file you mentioned works? Thanks and keep it up!

Since multiple are asking for a easier way of adding translations, I could possibly transfer all those translations to a json file. I'll look into possible solutions and reply back. :)

**Reply by DomHeadroom** [reply]
Have you ever tried to make the translation more straight forward by using weblate or crowdin?
So translator don't need to edit file directly but trough weblate/crowdin site and then their bot for github will do the rest for them.

Or at least give some instruction on the readme, because I want to to translate the app in Italian but I didn't find any clear instruction on how to do so, do I need to only edit the "Translations.kt" or create a yaml file or something else?

**Reply by zyrouge** [reply]
> Have you ever tried to make the translation more straight forward by using weblate or crowdin? So translator don't need to edit file directly but trough weblate/crowdin site and then their bot for github will do the rest for them.
> 
> Or at least give some instruction on the readme, because I want to to translate the app in Italian but I didn't find any clear instruction on how to do so, do I need to only edit the "Translations.kt" or create a yaml file or something else?

Just the YAML file. (Refer [this](https://github.com/zyrouge/symphony/wiki/Translations#adding-a-new-language)). Though I agree that using a service would be easier and would allow everyone.
